src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -64,46 +67,65 @@
-    private final FullQualifiedName MODEL_FQN       = new FullQualifiedName( "vehicle.model" );
-    private final UUID              studyEntitySetId;
-    private final UUID              deviceEntitySetId;
-    private final UUID              dataEntitySetId;
-    private final UUID              preProcessedEntitySetId;
-    private final UUID              recordedByEntitySetId;
-    private final UUID              usedByEntitySetId;
-    private final UUID              participatedInEntitySetId;
-    private final UUID              stringIdPropertyTypeId;
-    private final UUID              participantIdPropertyTypeId;
-    private final UUID              dateLoggedPropertyTypeId;
-    private final UUID              versionPropertyTypeId;
-    private final UUID              modelPropertyTypeId;
+    private final FullQualifiedName STRING_ID_FQN    = new FullQualifiedName( "general.stringid" );
+    private final FullQualifiedName PERSON_ID_FQN    = new FullQualifiedName( "nc.SubjectIdentification" );
+    private final FullQualifiedName DATE_LOGGED_FQN  = new FullQualifiedName( "ol.datelogged" );
+    private final FullQualifiedName STATUS_FQN       = new FullQualifiedName( "ol.status" );
+    private final FullQualifiedName VERSION_FQN      = new FullQualifiedName( "ol.version" );
+    private final FullQualifiedName MODEL_FQN        = new FullQualifiedName( "vehicle.model" );
+    private final FullQualifiedName DATE_USED_FQN    = new FullQualifiedName( "ol.datetime" );
+    private final FullQualifiedName START_DATE_TIME  = new FullQualifiedName( "ol.datetimestart" );
+    private final FullQualifiedName APP_PACKAGE_NAME = new FullQualifiedName( "general.fullname" );
+    private final FullQualifiedName APP_NAME         = new FullQualifiedName( "ol.title" );
+    private final FullQualifiedName RECORD_TYPE_FQN  = new FullQualifiedName( "ol.recordtype" );
+    private final FullQualifiedName DURATION         = new FullQualifiedName( "general.Duration" );
+    private final FullQualifiedName OL_ID_FQN        = new FullQualifiedName( "ol.id" );
+
+    private final UUID studyEntitySetId;
+    private final UUID deviceEntitySetId;
+    private final UUID dataEntitySetId;
+    private final UUID preProcessedEntitySetId;
+    private final UUID recordedByEntitySetId;
+    private final UUID usedByEntitySetId;
+    private final UUID participatedInEntitySetId;
+    private final UUID stringIdPropertyTypeId;
+    private final UUID personPropertyTypeId;
+    private final UUID dateLoggedPropertyTypeId;
+    private final UUID dateUsedPropertyTypeId;
+    private final UUID versionPropertyTypeId;
+    private final UUID modelPropertyTypeId;
+    private final UUID userAppsEntitySetId;
+    private final UUID appNamePropertyTypeId;
+    private final UUID appPackageNamePropertyTypeId;
+    private final UUID recordTypePropertyTypeId;
+    private final UUID startDateTimePropertyId;
+    private final UUID durationPropertyTypeId;
```

Comment:
How do you feel about switching to `ESID` and `PTID` instead of `EntitySetId` and `PropertyTypeId` to make these a bit shorter?

Reply:
Yeah, makes sense.